screen_brightness_control/__init__.py

```diff
@@ -17,6 +17,13 @@
 _logger = logging.getLogger(__name__)
 _logger.addHandler(logging.NullHandler())
 
+# Global variable to control whether duplicate monitors are allowed
```

Comment:
If this is refactored as a docstring under the variable, our documentation generator can pick it up for the online docs